File: Tareas/tarea8.rb
puts "¿Qué escala inicial desea escoger?" 
puts "1 para Celsius\n2 para Fahrenheit"
puts "\n"

escala = gets().to_i

case escala
    when 1
        puts "Ingrese los grados centígrados "
        n = gets().to_i
        temp = ((n * 1.8) + 32)
        puts "Grados Fahrenheit: #{temp}"

    when 2
        puts "Ingrese los grados fahrenheit"
        n = gets().to_i
        temp = ((n - 32) / 1.8)
        puts "Grados centígrados: #{temp}"

    else 
        puts "Ingresar una de las opciones válidas"
end

File: Tareas/tarea8.java
import javax.swing.*;

public class Tarea8 {
    public static void main(String[] args) {
        JOptionPane.showMessageDialog(null, "Azul Carolina Martínez Cabañas\n1868435 ITS\nPeriodo Enero-Junio 2020\nIng. Ismael Gutiérrez Pimentel");

        Integer escala = Integer.parseInt(JOptionPane.showInputDialog("Convertidor de temperaturas\n¿Qué escala inicial desea escoger?\n1 para Celsius\n2 para Fahrenheit"));

        switch (escala){
            case 1:
                Double nT = Double.parseDouble(JOptionPane.showInputDialog("Ingrese los grados centígrados "));
                Double tempF = ((nT * 1.8) + 32);
                JOptionPane.showMessageDialog(null, "Grados Fahrenheit: "+tempF);
                break;
            case 2:
                Double nF = Double.parseDouble(JOptionPane.showInputDialog("Ingrese los grados Fahrenheit"));
                Double tempC = ((nF - 32) / 1.8);
                JOptionPane.showMessageDialog(null,"Grados centígrados: " + tempC);
                break;
            default:
                JOptionPane.showMessageDialog(null, "Ingresar una opción válida");
            }
        }
}